tdrs-frontend/README.md

```diff
@@ -17,90 +17,26 @@ Frontend API Service for TDP. Deployed to Cloud.gov at https://tdp-frontend.app.
 - [Unit and Integration Testing](#Unit-and-Integration-Testing)
 - [Manual Cloud.gov Deployments](#Manual-Cloud.gov-Deployments)
 
-# Testing the local Frontend Service:
+# Running the Frontend locally:
 
-  **Login is now dependent on the [tdrs-backend](../tdrs-backend/README.md) service. You will need a local instance of that application running.**
+  **Login is now dependent on the [tdrs-backend](../tdrs-backend/README.md#testing-the-local-backend-service) service. You will need a local instance of that application running.**
 
+### Local Development:
 
-### Local Development Options
+Execute the command below from `tdrs-frontend/` folder to access the frontend at `http://localhost:3000`
 
-**Commands are to be executed from within the `tdrs-frontend` directory**
-
-1.) Create a `.env.local` file for local development configured to reference the backend service:
-
-```bash
-$ touch .env.local && echo "REACT_APP_BACKEND_URL=http://localhost:8080/v1" >> .env.local
 ```
- 
-  - The `REACT_APP_BACKEND_URL` variable in this file points to the backend host. For local testing this value should default to the following :
-  
-   ```
-   http://localhost:8080/v1
-   ```
-
-
-2.) Frontend project spin-up options: 
-
-- Option 1 (Using Yarn directly): We recommend using [NVM](https://github.com/nvm-sh/nvm)
```

Comment:
I like that this removes the section about installation via yarn, since I like having a single set of instructions for building the app (Docker) that matches how the app is built in production (Docker).

However, FWIW the latest I'm seeing from @hilvitzs in Teams (and seeing myself locally) is that local development using Docker is broken and only local development using yarn works right now.

I don't thing we necessarily need to change anything about this PR, but I do think this underscores the importance of fixing the local environment variable setups on both the backend and the frontend.